_data/simple-icons.json

```diff
@@ -20,6 +20,11 @@
             "hex": "000000",
             "source": "https://doc.e.foundation"
         },
+        {
+            "title": "1&1",
+            "hex": "003D8F",
+            "source": "https://www.united-internet.de/en/newsroom/media-center/logos.html"
+        },
```

Comment:
Looks like the linter is complaining about the order here, try moving this down below 1.1.1.1?